api/external/compliance/reporting/reporting.proto

```diff
@@ -655,6 +655,20 @@ message Control {
 	repeated Ref refs = 8;
 	// Metadata defined on the control in key-value format.
 	map<string, string> tags = 9;
+	// Indicates if the control has been waived or not
+	string waiver_str = 10;
```

Comment:
@alexpop i think the mismatch here is what's causing all the failures -- here waiver_str is 10 and orig data is 11; on the internal proto waiver_str is 11 and orig data is 12 -- mismatches there will cause the api requests to fail